eng/pipelines/update-images-pipeline.yml

```diff
@@ -73,8 +73,10 @@ extends:
 
               - pwsh: |
                   go run ./cmd/dockerupdatepr `
```

Comment:
@dagood is this pipeline actually being used anymore? When I checked here it hasn't been run in some time https://dev.azure.com/dnceng/internal/_build?definitionId=1040&view=runs